test/lib/mayaUsd/render/mayaToHydra/testViewportFilters.py

```diff
@@ -222,5 +231,99 @@ def test_Strokes(self):
 
     # TODO : Texture Placements (not working in Hydra as of 2024-05-03)
 
+    # --- USD data ---
+
+    def test_UsdPolygons(self):
+        from pxr import UsdGeom
+
+        stagePath = mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+        stage = mayaUsd.lib.GetPrim(stagePath).GetStage()
+
+        capsuleName = "UsdCapsule"
+        capsuleXform = UsdGeom.Xform.Define(stage, "/" + capsuleName + "Xform")
+        capsuleXform.AddTranslateOp().Set(value=(6, 0, 0))
+        UsdGeom.Capsule.Define(stage, str(capsuleXform.GetPath()) + "/" + capsuleName)
+        cmds.select(clear=True)
```

Comment:
Very minor, but why do we need to clear the selection repeatedly?

Reply:
No reason. Updated to keep only one before the image diffing.